app/views/shared/_youtube_player.html.erb

```diff
@@ -25,15 +25,16 @@
         </span>
         <br>
         <%= no_cookies_or_text %>
-      </p>
       <a
         href="<%= youtube_video_url %>"
         class="govuk-body govuk-link"
         data-youtube-player-analytics="true"
         data-youtube-player-analytics-category="EmbeddedYoutube"
       >
+        <br>
```

Comment:
Nitpick: while it's valid HTML to have `<br>` as a child of `<a>`, I'd probably move it to right before the `<a>` in this particular instance.
It just slightly bothers me that in some browsers when you right-click the link the highlight looks slightly odd due to the `<br>`. 😶

**On Chrome**
<img width="744" alt="Screenshot 2020-10-20 at 12 00 52" src="https://user-images.githubusercontent.com/7116819/96577840-2ba82400-12cc-11eb-9a86-8029dd707036.png">


**On Safari**
<img width="682" alt="Screenshot 2020-10-20 at 12 01 36" src="https://user-images.githubusercontent.com/7116819/96577864-3498f580-12cc-11eb-8165-3684b0cc9fb3.png">

Reply:
Agreed, does look much better - updated.